include/behaviortree_cpp_v3/xml_parsing.h

```diff
@@ -36,9 +36,6 @@ class XMLParser : public Parser
   Pimpl* _p;
 };
 
-void VerifyXML(const std::string& xml_text,
```

Comment:
I think you should not remove this from the public interface

Reply:
Understood, I put it back. Good to know that Groot uses this too.